tests/test_codebase/test_mmdet3d/test_mmdet3d_models.py

```diff
@@ -107,3 +107,53 @@ def test_pointpillars_scatter(backend_type: Backend):
             rewrite_output = rewrite_output.cpu().numpy()
         assert np.allclose(
             model_output.shape, rewrite_output.shape, rtol=1e-03, atol=1e-03)
+
+
+def get_centerpoint():
+    from mmdet3d.models.detectors.centerpoint import CenterPoint
+    model_cfg = load_config('data/model_cfg.py')[0]
```

Comment:
use config_path relative to `MMDEPLOY_ROOT`, otherwise this would fail to find file `data/model_cfg.py`